pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -63,6 +58,12 @@ def work(self):
                                         pokemon['pokemon_data'][individual_stat] = 0
                                         continue
 
+                                iv_display = '{}/{}/{}'.format(
```

Comment:
Yes, the individual stats are set to 0 (line 58) if they key isn't found in the dict.